crates/node-core/src/version.rs

```diff
@@ -11,6 +16,35 @@ pub const VERGEN_GIT_SHA: &str = env!("VERGEN_GIT_SHA");
 /// The build timestamp.
 pub const VERGEN_BUILD_TIMESTAMP: &str = env!("VERGEN_BUILD_TIMESTAMP");
 
+//The identification of the client
+/// - The two letter client code
+/// - The human readable name of the client
+/// - The version string of the current implementation 
+/// - first four bytes of the latest commit hash of this build
+/// 
+/// # Example
+/// 
+///  ```text
+/// code: Rh
+/// name: reth
+/// version: 0.1.0
+/// commit: defa64b2
+///  ```
+pub const CLIENTVERSIONV1: &str = const_str::concat(
```

Comment:
this should be a struct, the spec expects us to send a json object over the wire that looks like

```js
{
  code: "RH",
  name: "Reth",
  version: "v0.2.0-beta.5",
  commit: "defa64b2"
}
```